util/topgen.py

```diff
@@ -390,7 +390,8 @@ def generate_pwrmgr(top: Dict[str, object], out_path: Path) -> None:
         "Wkups": top["wakeups"],
         "rst_reqs": top["reset_requests"],
         "NumRstReqs": n_rstreqs,
-        "wait_for_external_reset": top['power']['wait_for_external_reset']
+        "wait_for_external_reset": top['power']['wait_for_external_reset'],
+        "NumRomInputs": int(top['power']['NumRomInputs'])
```

Comment:
This seems redundant: should it be derived from the number of modules with 'type' == 'rom_ctrl'?

Reply:
Thanks. Used `lib.num_rom_ctrl()` for that.